src/gui/app.cpp

```diff
@@ -13,6 +13,65 @@
 
 namespace NeovimQt {
 
+struct ConnectorInitArgs {
+	enum Type {
+		Embed,
+		Server,
+		Spawn,
+		Default
+	};
+
+	const Type type;
+	const int timeout;
```

Comment:
Provide a reasonable default value:
`const int timeout{ ??? };`

Necessary for primitive types.

Reply:
Isn't the default value always provided by the command line parser?

Reply:
Yes. Your code is logically sound.

This is about writing future-proof code. Primitive types such as `int` will not be zero initialized. Providing a default value helps future authors from accidentally introducing non-deterministic behavior. Hard to catch, hard to debug.